protocol/app/upgrades/v6.0.0/upgrade.go

```diff
@@ -42,6 +49,56 @@ func removeStatefulFOKOrders(ctx sdk.Context, k clobtypes.ClobKeeper) {
 	}
 }
 
+func setMarketMapParams(ctx sdk.Context, mmk marketmapkeeper.Keeper) {
+	err := mmk.SetParams(ctx, marketmaptypes.Params{
+		// todo fill out these fields
+		MarketAuthorities: nil,
+		Admin:             "",
+	})
+	if err != nil {
+		panic(fmt.Sprintf("failed to set x/mm params %v", err))
+	}
+}
+
+func migratePricesToMarketMap(ctx sdk.Context, pk pricestypes.PricesKeeper, mmk marketmapkeeper.Keeper) {
+	h, err := dydx.NewAPIHandler(zap.NewNop(), config.APIConfig{
+		Enabled:          true,
+		Timeout:          1,
+		Interval:         1,
+		ReconnectTimeout: 1,
+		MaxQueries:       1,
+		Atomic:           false,
+		Endpoints:        []config.Endpoint{{URL: "upgrade"}},
+		BatchSize:        0,
+		Name:             dydx.Name,
+	})
```

Comment:
Why do we need this? It doesn't look like the `ConvertMarketParamsToMarketMap` method requires anything stateful from the handler and can just be a stateless function. I think it'd be better not to have this dependency if it's not needed

Reply:
Left a comment.  We just need this to init the handler.  Refactoring the handler on our end at some point makes perhaps.  But right now its coupled to our handlers by way of logging mainly.